### PR TITLE
MVP; add personal info, projects, contact info, and random fact button.

### PR DESCRIPTION
**Pull request by kayefung**

### CODE REVIEW

**Comment by kayefung** [comment]
Before screenshot:

before_pr2

After:

after_pr2

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,52 @@
...
+          </ul>
+          <li>Weighted Set Cover Problem Solver</li>
+          <ul>
+              <li>Implemented three greedy algorithms to solve the weighted set cover problem</li>
+          </ul>
+          <li>Vector and Matrix Computation Package</li>
+          <ul>
+              <li>Created a linear algebra package that allowed for vector and matrix computations such as
+              multiplication and transpositions</li>
+          </ul>
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/index.html:46]: Inconsistent indentation.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,17 @@
...
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I\'m a middle child!', 'I\'m a cat person!', 'One time I flushed a pair of sunglasses down a toilet:(',
+        'I can play the ukulele!'];
 
   // Pick a random greeting.
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/script.js:23]: Comment doesn't reflect new method. Should say fact.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,52 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Kaye's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="setBackgroundColor()">
```
Comment (ajpaon) [inline portfolio/src/main/webapp/index.html:9]: where is this defined?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,52 @@
...
+      <p>At school I've volunteered as an instructor with Project Include, providing free coding bootcamps
+          to underprivileged school-aged kids around Toronto. Additionally I've had a hand in planning the
+          Engineering Faculty's Orientation Week, being responsible for Registration and Downtown
+          Walkaround (a 1000+ person parade around Downtown Toronto).</p>
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">Hello</button>
+      <div id="fact-container"></div>
+      <h2>Projects</h2>
+      <ul>
+          <li>Information Retrieval Search Engine</li>
```
Comment (juanalonsocruz) [inline portfolio/src/main/webapp/index.html:28]: Still has indentation issue. Between the top level \<ul> and each second level \<li> as well as the third level \<ul> and fourth level \<li>. Won't block merge but please fix.

**Pull request merged**